config/credentials/production.key

```diff
@@ -0,0 +1 @@
+9cc4f74f9e9d5c66061d6592c035a353
```

Comment:
Request for change: Should this be here, unencrypted? What is this used for?

Reply:
Unused
Removed. CiricleCI is using config/master.key (gitignore)